src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala

```diff
@@ -130,6 +133,20 @@ class ProxyService(proxyConfig: ProxyConfig,
     }
   }
 
+  private def logRequestForMetrics(userInfo: UserInfo, request: HttpRequest, response: HttpResponse) = {
+    val headers = request.headers
+    val headerMap: Map[String, String] = headers.map{ header =>
+      (header.name(),header.value())
+    }.toMap
+
+    logger.info(s"${headerMap.getOrElse("X-Forwarded-For", "0.0.0.0")} ${userInfo.userEmail} " +
```

Comment:
I am wondering if the written will be in the right format. Our logback logger formats logs like so: https://github.com/DataBiosphere/leonardo/blob/develop/src/main/resources/logback.xml#L23

I wonder if we should have a separate `Logger` instance for access logs vs application logs.